### PR TITLE
Fix ineffassign

### PR DESCRIPTION
**Pull request by haoshuwei**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix some files which have inefficient assign statements.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

Signed-off-by: haoshuwei <haoshuwei24@gmail.com>

### CODE REVIEW

**Comment by deads2k** [comment]
reflects reality
/approve
/ok-to-test

File: pkg/volume/csi/csi_plugin_test.go [hunk 1]
```diff
@@ -620,6 +620,9 @@ func TestPluginNewMounter(t *testing.T) {
 			if csiClient == nil {
 				t.Error("mounter csiClient is nil")
 			}
+			if err != nil {
```
Comment (jpbetz) [inline pkg/volume/csi/csi_plugin_test.go]: Looks like error is already handled correctly above in L593-599 to account for `test.shouldFail`?

File: staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer_test.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func TestInputStreamReader(t *testing.T) {
 		return
 	}
 	defer readCloser.Close()
-	result, err := ioutil.ReadAll(readCloser)
+	result, _ := ioutil.ReadAll(readCloser)
```
Comment (jpbetz) [inline staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer_test.go]: `t.Fatal` if this error is not nil?

File: staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer_test.go [hunk 3]
```diff
@@ -118,7 +118,7 @@ func TestInputStreamTransport(t *testing.T) {
...
```
Comment (jpbetz) [inline staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer_test.go]: `t.Fatal` if this error is not nil?

**Review by jpbetz — COMMENTED** [review]
A couple comments, looks good otherwise.

**Pull request merged**